content/en/blog/2019/webhook/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Secure webhook management
+description: A more secure way to manage Istio webhooks.
+publishdate: 2019-10-29
```

Comment:
Should the date of both of your blogs be changed to align with the release date?

Reply:
Done.